src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs

```diff
@@ -110,6 +116,11 @@ public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, s
                     queryStringParams.Add("format", format);
                 }
 
+                if (appendVersion?.Equals("true", StringComparison.InvariantCultureIgnoreCase) == true)
```

Comment:
I don't think you need the invariant check here.

Reply:
We need some culture config here. Since it's coming from a user-provided string, the invariant culture seems appropriate to me. Otherwise, ordinal, since it's a technical string.

Reply:
Sure.